sdk/framework/backend.go

```diff
@@ -486,9 +486,18 @@ func (b *Backend) handleRootHelp() (*logical.Response, error) {
 		return nil, err
 	}
 
+	// The plugin type (e.g. "kv", "cubbyhole") is only assigned at the time
+	// the plugin is enabled (mounted). If specified in the request, the type
+	// will be used as part of the request/response body names in the OAS
+	// document
+	var requestResponsePrefix string
```

Comment:
Is there a possibility of this variable staying as an empty string? And if so will that cause any problems with the spec generation?

Reply:
Yes, it is possible if whoever calls the plugin's /help endpoint does not specify the "requestResponsePrefix". In this case, the generated request name will be missing that prefix. For example "KvLookupRequest" would be replaced with just "LookupRequest". I'm not really sure what the best way of dealing with this is (or if it should be of concern). In the OpenAPI generation workflow, the prefix will always be set.